tests/unittests/Foundation/NSFileHandleTestFile.txt

```diff
@@ -1 +0,0 @@
-The Quick Brown Fox.
```

Comment:
hmm i don't think this is being used :0

---
In reply to: [111624327](https://github.com/Microsoft/WinObjC/pull/2512#discussion_r111624327) [](ancestors = 111624327)